eslint.config.mjs

```diff
@@ -0,0 +1,36 @@
+import hmppsConfig from '@ministryofjustice/eslint-config-hmpps'
+
+export default [
+  // shared HMPPS standards
+  ...hmppsConfig,
+  
+  // app-specific overrides
+  {
+    ignores: ['**/node_modules', 'dist/', 'test_results'],
```

Comment:
had to extend as playwright dashboards get written to `test_results` 

This is actually a change I could push to the template project, I tidied up the reporting folders which simplifies circle ci config too

<img width="209" alt="Screenshot 2024-11-07 at 15 02 37" src="https://github.com/user-attachments/assets/47cf4111-d1fe-47ee-b55c-3b7b30e9fbf4">